last_seen.py

```diff
@@ -0,0 +1,27 @@
+from brutal.core.plugin import BotPlugin, cmd, event
+from time import strftime, gmtime
+
+class LastSeen(BotPlugin):
+    def setup(self, *args, **kwargs):
+        self.seen = self.open_storage('last_seen')
+
+    @event
+    def seen_evt(self, event):
+        events = ['quit', 'part', 'kick', 'join', 'rename', 'topic']
+        if event.event_type in events:
+            nick = event.meta['nick']
+            time = strftime("%H:%M %d.%m.%Y", gmtime())
+            self.seen[nick] = time
```

Comment:
@JakubNvk please do not let your biased view of a small channel fool you. Many users use bouncers and other methods of staying always online. In a huge channel where there are many simultanious discussions one can get lost pretty easily. Thus, it is indeed necessary in my opinion to provide the user with as much context as possible and that definitely includes the event.